exporters/jaeger/src/main/java/io/opentelemetry/exporters/jaeger/JaegerGrpcSpanExporter.java

```diff
@@ -223,27 +251,12 @@ public static Builder fromEnv() {
      * @return a new exporter's instance.
      */
     public JaegerGrpcSpanExporter build() {
+      if (channel == null) {
```

Comment:
Think we should validate that endpoint is set to provide a better error message, guessing otherwise it's an NPE.

Reply:
oh, i just realized, endpoint has a default, so no need to validate here i think?